Server/Server.Core/Server.Core.Static/KeysValidator.hpp

```diff
@@ -16,7 +16,7 @@ class KeysValidator
 private:
     struct Keys
     {
-        const std::string realDB = "-p";
+        const std::string withPort = "-p";
```

Comment:
Example from psql application. This flag indicate that the port is in used and its value is XXXX. Make more sens in the variable name.
""" text 
...
-p, --port=PORT          database server port (default: "5432")
...
"""